frontend/app/styles/app.css

```diff
@@ -33,10 +33,36 @@ li a {
   resize: vertical;
 }
 
-#accountFormModalLabel {
-  display:  inline;
+.modal-title {
+  display: inline;
 }
 
 #team-power-select {
   height: 69px;
 }
+
+.private-description{
+  background-color: #eee;
+  position: absolute;
+  padding: 2px;
+  margin-left: 4px;
+}
+
+.info-icon{
+  width: 18px;
+  height: 18px;
+  display: inline;
+}
+
+.info-icon:hover{
+  cursor: help;
+}
+
+#checkbox-private {
+  margin-left: 5px;
+}
+
+.nav .open > a, .nav .open > a:hover, .nav .open > a:focus {
+  background-color: #2988bc !important;
```

Comment:
To fix strange behaviour in the Add-Dropdown in the NavBar. Otherwise, when opening the dropdown, the Arrow and Default button get white.